espnet/nets/chainer_backend/deterministic_embed_id.py

```diff
@@ -146,25 +145,30 @@ def backward(self, indexes, grads):
 
 
 def embed_id(x, W, ignore_label=None):
-    """Efficient linear function for one-hot input.
+    r"""Efficient linear function for one-hot input.
```

Comment:
Why is this part treated as raw? it will not affect the docstring?

Reply:
I actually write as `"""Efficient ....` at the first time. However, the `flake8-docstrings` tool tells me as follows.

`deterministic_embed_id.py:148:1: D301 Use r""" if any backslashes in a docstring`

So I add `r` before `"""`.

Reply:
I am wondering where is the `flake8-docstrings` test implemented. I checked in the `CI` folder and others and the only test implemented is `flake8`. I tested this code with `flake8` and could not have the error mentioned. Also, when I tried installing `flake8-doctrings` showed me this message:

```
"flake8-docstrings" failed during execution due to "module 'pydocstyle' has no attribute 'tokenize_open'"
Run flake8 with greater verbosity to see more details
```
If the docs should be tested with `flake8-docstrings` should not be this also included in the 'CI/doc.sh' file?

Reply:
* You are right, I checked the `CI` folder and found that the `flake8-docstrings` is not used. I referenced #894 and used `flake8-docstrings` to check the modified docstrings on my local environment. I don't know if I should not use `flake8-docstrings`, so I will ask @kan-bayashi about the tool to check the docstrings.
* I installed `flake8-docstrings` again but I didn't get the message above. Try to install the `flake8-docstrings` as mentioned in #894

Reply:
@Fhrozen Currently we do not introduce flake8-docstrings because there are too many warning and error to modify. This PR is the first step to introduce `flake8-docstrings` in CI.